source/guides/references/protractor-migration-guide.md

```diff
@@ -0,0 +1,565 @@
+
+**Before: Protractor**
+
+```javascript
+const fs = require('fs')
+
+describe('Dashboard', () => {
+  it('should render dashboard', () => {
+    browser.get('/dashboard')
+    browser.takeScreenshot().then((png) => {
+      const stream = fs.createWriteStream(filename)
+
+      writeScreenShot(png, 'dashboard.png')
+      stream.write(new Buffer(png, 'base64'))
+      stream.end()
+    })
+  })
+})
+```
+
+**After: Cypress**
+
+```javascript
+describe('Dashboard', () => {
+  it('should render dashboard', () => {
+	  cy.visit('/dashboard')
+	  cy.screenshot('dashboard.png')
+  })
+})
+```
+
+For more information on what configuration options are available, be sure to check out the [official Cypress documentation on the screenshot command](https://docs.cypress.io/api/commands/screenshot.html).
+
+## FAQs
+
+### Do I have to replace all of my tests with Cypress immediately?
+
+Absolutely not. While it might sound ideal to replace Protractor entirely, we know that most projects and teams still need to deliver features to customers and it's unrealistic to halt the team's progress.
+
+### Can Protractor and Cypress coexist in the same app?
+
+Yes! Your Protractor tests would continue to live in the `e2e` directory that Angular CLI scaffolded while all Cypress tests would live in a sibling folder named `cypress`.
+
+```
+.
+├── cypress
+├── e2e
+├── src
+├── .editorconfig
+├── .gitignore
+├── angular.json
+├── browserslist
+├── cypress.json
+├── karma.conf.js
+├── package.json
+├── README.md
+├── tsconfig.app.json
+├── tsconfig.json
+├── tsconfig.spec.json
+└── tslint.json
+```
+
+In fact, as you work through migrating to Cypress, we believe that progressively enhancing your e2e tests with Cypress is the best path forward to ensure that feature development is not impacted.
+
+## Getting Started
+
+Like any library you would add to your application, we start by adding it as a dependency to our `package.json`.
+
+```
+npm install cypress -D
+```
+
+Then, since Cypress can run in parallel with your application, let's install Concurrently to simplify our npm script.
+
+```
+npm install concurrently -D
+```
+
+Then we will update our `package.json` with the a script to run Cypress and your Angular app simultaneously:
+
+```js
+"cypress": "concurrently \"ng serve\" \"cypress open\""
+```
+
+```js
+// Example config in package.json
+{
+  "scripts": {
+    ...
+    "e2e": "ng e2e",
+    "e2e:debug": "node --inspect-brk ./node_modules/.bin/protractor ./e2e/protractor.conf.js",
+    "cypress": "concurrently \"ng serve\" \"cypress open\""
+  },
+  "dependencies": { ... },
+  "devDependencies": { ... }
+}
+```
+
+Now, when we run:
+
+```bash
+npm run cypress
+```
+
+It will start up Cypress and our Angular app at the same time!
+
+{% video local /img/snippets/npm-run-cypress.mp4 %}
+
+## Essentials
```

Comment:
make this a main section heading too - need to update all subheadings to be h2 and h3 respectively (this will break things if you don't update subheadings! so make sure to update)
 
```suggestion
# Essentials
```